packages/email-service/test/app.e2e-spec.ts

```diff
@@ -33,17 +34,17 @@ async function initApp() {
 beforeAll(async () => {
   app = await initApp();
 
-  const proto = ProtoLoader.loadSync([ResetProtoFile]) as any;
+  const proto = ProtoLoader.loadSync([EmailProtoFile]) as any;
 
   const protoGRPC = GRPC.loadPackageDefinition(proto) as any;
 
-  const resetClient = new protoGRPC.juno.reset_db.DatabaseReset(
+  const emailClient = new protoGRPC.juno.emailService(
     process.env.DB_SERVICE_ADDR,
     GRPC.credentials.createInsecure(),
   );
 
   await new Promise((resolve) => {
-    resetClient.resetDb({}, () => {
```

Comment:
These lines should not be changed, this resets the database state so the tests in the file run independent of other tests. Add the lines you changed adding the emailClient variable below these